src/settings.rs

```diff
@@ -1,41 +1,29 @@
-use regex::Regex;
-use serde::{Deserialize, Serialize};
 use std::collections::HashSet;
 
-#[derive(Clone, Serialize, Deserialize, Default, Debug, PartialEq, Eq, Hash)]
-#[serde(rename_all = "camelCase")]
-#[allow(clippy::enum_variant_names)] // allow for all to end in `In`
-pub(crate) struct EnvVar {
-    pub name: String,
-    pub value: Option<String>,
-}
-
-#[derive(Serialize, Deserialize, Default, Debug, PartialEq, Eq)]
-#[serde(rename_all = "camelCase")]
-#[allow(clippy::enum_variant_names)] // allow for all to end in `In`
-pub enum Operator {
-    AllAreUsed,
-    NotAllAreUsed,
-    #[default]
-    AnyIn,
-    AnyNotIn,
-}
-
-#[derive(Serialize, Deserialize, Default, Debug)]
-#[serde(default)]
-#[serde(rename_all = "camelCase")]
-pub(crate) struct Rule {
-    pub reject: Operator,
-    pub environment_variables: HashSet<EnvVar>,
-}
+use regex::Regex;
+use serde::{Deserialize, Serialize};
 
 // Describe the settings your policy expects when
 // loaded by the policy server.
```

Comment:
This is the default from the template, we should either remove it or put something appropriate